project_event/models/project_task.py

```diff
@@ -135,11 +135,33 @@ class Task(models.Model):
         default='draft',
         track_visibility='onchange',
     )
-    task_state_report_done_required = fields.Selection(
-        related='task_state'
+    task_state_report_done_required = fields.Selection([
```

Comment:
@eilst  Two fields have the same structure except for the computing methods.  it seem like the code redundant